### DISCUSSION TITLE
Talentcell

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#20] [Ideas]
We should tell people using a talentcell to put a kill switch on both the 5v and 12v hot side of the battery to the pack. When charging a talentcell, it makes you leave the battery on to charge.   Which could cause issues with the pack down the line being left on for hours on end charging. Just an idea.  I did it to mine on the hot side of each power source.   Ground won't work as the pack still received ground from other circuits

**Comment by gpstar81** [comment]
Good idea.

**Comment by DustinGrau** [comment]
Adding content to the POWER guide now to explain the "DPDT Mod" for use with a charging port. Will be out with the v5.3.4 release
